app/src/main/java/com/github/fribourgsdp/radio/LobbyActivity.kt

```diff
@@ -192,7 +217,7 @@ open class LobbyActivity : AppCompatActivity() {
 
                 val isGameLaunched = snapshot.getBoolean("launched")
 
-                if (!isHost && isGameLaunched != null && isGameLaunched) {
+                if (!isHost && isGameLaunched != null && isGameLaunched && hasVoiceIdPermissions) {
```

Comment:
Here instead of just not doing anything, it would be better to display an error when `hasVoiceIdPermissions` is false. The other conditions of this if-statement do not trigger an error as it is made to redirect players.

Reply:
You are right. What we wanted to implement as a feature for next week is that the host cannot launch a lobby until all the players have the appropriate permissions. For now we just wanted to make sure that a user wouldn't try initializing a voice channel and joining it without having the appropriate permissions.

Reply:
Ok it is even better this way. All good if you do it next week. 👍🏻